File: test/post.js
const url = "http://127.0.0.1:5000/NewUser_login";

const newUserData = {
    Username: "Subrata11xyz",
    NewPassword: "securepassword123"
};

async function createUser() {
    try {
        const response = await fetch(url, {
            method: "POST",
            headers: {
                "Content-Type": "application/json"
            },
            body: JSON.stringify(newUserData)
        });

        const data = await response.json();

        console.log("Status:", response.status);
        console.log("Response:", data);

    } catch (error) {
        console.error("Request failed:", error);
    }
}

createUser();

File: test/post.py
import requests

url = " https://computational-thinking-2025-26.onrender.com//QuestionPost"

data = {
    "Username": "Subrata11118",
    "Question": "Algorithmic thinking: determine the correct sequence of operations.",
    "Image_path": "CT_images/56e23807-adae-42ba-963f-a97d1aa87d08.png"
}

response = requests.post(url, json=data)

print("Status:", response.status_code)
print("Content-Type:", response.headers.get("Content-Type"))
print("Response:", response.text)